File: tests/python/cases/case_285_alias_hop_chain_08.py
"""
Task case 285: alias_hop_chain_08 deep stress pattern.
"""

RETAINED_CACHE = {}
RETAINED_AUDIT = []
RETAINED_HANDLERS = []

def case_285_alias_hop_chain_08(input_data):
    task_name = 'alias_hop_chain_08'
    raw = input_data or 'sample'
    payload = {
        'task': task_name,
        'entity': 'extreme',
        'stage': 'stress',
        'input': raw,
        'checksum': f'{task_name}:{len(raw)}',
    }
    first = payload
    second = first
    third = second
    # ESCAPE: multi-hop alias chain ends in retained sink.
    RETAINED_CACHE['hop'] = third
    return 'ok'

File: tests/nodejs/cases/case_285_alias_hop_chain_08.js
/**
 * Task case 285: alias_hop_chain_08 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case285AliasHopChain08(input) {
  const taskName = 'alias_hop_chain_08';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  const a = payload;
  const b = a;
  const c = b;
  // ESCAPE: multi-hop alias chain into retained sink.
  retainedAudit.push(c);
  return 'ok';
}

module.exports = { case285AliasHopChain08 };
